scripts/start_tiup.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 set -ex
 tidb_version=$1
-mode=$2
+without_ngm=${2:-false}
+mode=${3:-"start"}
```

Comment:
The order of args will cause failing to restart TiUP.

Reply:
Nice catch! Thanks!